### DISCUSSION TITLE
Add posibility to add custom block to Generator

### DISCUSSION THREAD

**Discussion by SwartZCoding** [BG-Software-LLC/SuperiorSkyblock2#1198] [Implemented Feature Requests] [answered]
All in the title :)

Add the possibility to add custom block on generator by using ItemsAdder or Oraxen. Good idea for Custom ores.

**Comment by OmerBenGera** [comment, marked as answer]
Oraxen is already supported by adding "ORAXEN:<BLOCK>" as a block.

**Reply by SwartZCoding** [reply]
> Oraxen is already supported by adding "ORAXEN:" as a block.

Ok and for ItemsAdder users ? :)

**Reply by OmerBenGera** [reply]
> > Oraxen is already supported by adding "ORAXEN:" as a block.
> 
> Ok and for ItemsAdder users ? :)

Not a feature yet, thats why I kept this opened.

**Comment by SwartZCoding** [comment]
Hello ! Just for up this thread. I think Itemsadder is now a very important feature to add ... He is most famous than Oraxen. Lots of people wait this feature !

**Comment by Hysteriox** [comment]
+1

**Comment by NewZAZ** [comment]
+1

**Comment by OmerBenGera** [comment]
Already a thing.
Use "ITEMS_ADDER:" as a prefix.
